packs/hpe-icsp/actions/icsp_server_data_format.py

```diff
@@ -17,7 +17,19 @@
 
 
 class FormatServerData(ICSPBaseActions):
-    def run(self, mids, hostnames, domains=None, workgroups=None):
+    def run(self, identifiers, identifier_type, hostnames,
+            domains=None, workgroups=None, connection_details=None):
+        if identifier_type == "mid":
+            for n in identifiers:
+                try:
```

Comment:
Would be nice to refactor this in a separate utility validation function, but it's not such a big deal.